internal/gengapic/paging.go

```diff
@@ -278,6 +315,13 @@ func (g *generator) pagingCall(servName string, m *descriptor.MethodDescriptorPr
 
 func (g *generator) pagingIter(pt *iterType) {
 	p := g.printf
+	if pt.mapValueTypeName != "" {
+		p("// Holder type for string/%s map entries", pt.mapValueTypeName)
```

Comment:
Comments for an exported element should begin with the name of that element e.g. `// FooPair is a holder type...`